languages/rust/exercises/concept/structs/.docs/instructions.md

```diff
@@ -0,0 +1,39 @@
+# Instructions
+
+In this exercise you'll be implementing methods on a `struct`. 
```

Comment:
`instructions.md` document should [consist of two parts](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsinstructionsmd):

1. The story/theme of the exercise
1. The tasks the students has to solve

Currently, this exercise does not have a story theme. The theme itself does not have to be complex, it can be [simple](https://github.com/exercism/v3/blob/master/languages/rust/exercises/concept/numbers/.docs/instructions.md). Whatever works best for the exercise.

Similarly, this exercise does not yet explicitly list the individual tasks, which you can see an example of [here](https://github.com/exercism/v3/blob/master/languages/rust/exercises/concept/numbers/.docs/instructions.md).

Reply:
Theme and tasks suggestion:

> It is often useful to group a collection of items together, and handle those groups as units. In Rust, we call such a group a struct, and each item one of the struct's fields. A struct defines the general set of fields available, but a particular example of a struct is called an instance.
> 
> Furthermore, structs can have methods defined on them, which have access to the fields. The struct itself in that case is called `self`. When a method uses `&mut self`, the fields can be changed, or mutated. When a method uses `&self`, the fields cannot be changed: they are immutable. Controlling mutability helps the borrow-checker ensure that entire classes of concurrency bug just don't happen in Rust. 
> 
> In this exercise, you'll be implementing two kinds of methods on a struct. The first are generally known as getters: they expose the struct's fields to the world, without letting anyone else mutate that value. In Rust, these methods idiomatically share the name of the field they expose.
>
> You'll also be implementing methods of another type, generally known as setters. These change the value of the field. Setters aren't very common in Rust--if a field can be freely modified, it is more common to just make it public--but they're useful if updating the field should have side effects, or for access control: a `pub(crate)` setter allows other modules within the same crate to update a private field, which can't be affected by the outside world.